### PR TITLE
Fix trace tab styles

### LINKED ISSUE #9631: [tracing] Incorrect visualization of tooltip
### Describe the bug

Image

- Tooltip data is not readable
- When scrolling, the graph goes outside the margin
- Trace disappears when click on the Span Details tab

Same with Jaeger

### What are the steps to reproduce this bug?

1. `istio/tempo/install-tempo-env.sh -c kubectl -ik true`
2. Go to any trace
3. …

### Environment
*Learn about how to determine versions [here](https://kiali.io/docs/faq/general/#how-do-i-determine-what-version-i-am-running).*

- **Kiali version:**
- **Istio version:**
- **Kubernetes impl:** 
- **Kubernetes version:** 
- **Other notable environmental factors:**

https://github.com/user-attachments/assets/6e8fc137-ac07-4e33-a353-0e832c3a3c78

### PR DESCRIPTION
**Pull request by josunect**
### Describe the change

- Fix text in tracing tooltip
- Fix scroll in Scatter chart
- Fix tracing tooltip hidden issue in the scatter chart section. Changing position
image

- Fix issue when click in "Span Details" made disapear the Scatter tooltip

image

### Steps to test the PR

- Go to the traces tab and review fixed issues

### Automation testing

If applicable, explain the case scencarios covered by **unit / integration / e2e** tests created for this PR.

### Issue reference

Fixes https://github.com/kiali/kiali/issues/9631

### CODE REVIEW

File: frontend/src/components/TracingIntegration/TracesComponent.tsx [hunk 1]
```diff
@@ -358,11 +357,7 @@ class TracesComp extends React.Component<TracesProps, TracesState> {
             </CardBody>
           </Card>
           {this.props.selectedTrace && (
-            <div
-              style={{
-                marginTop: 25
-              }}
-            >
+            <div style={{ marginTop: 25 }}>
```
Comment (ferhoyos) [inline frontend/src/components/TracingIntegration/TracesComponent.tsx]: Not changed in this PR, but it is recommended to be in rem (2rem)
```suggestion
            <div style={{ marginTop: '2rem' }}>
```

File: frontend/src/components/TracingIntegration/TraceTooltip.tsx [hunk 2]
```diff
@@ -12,26 +12,45 @@ import { PFColors } from 'components/Pf/PfColors';
 import { HookedChartTooltip, HookedTooltipProps } from 'components/Charts/CustomTooltip';
 import { formatDuration } from 'utils/tracing/TracingHelper';
 
-const flyoutWidth = 280;
-const flyoutHeight = 130;
+const flyoutWidth = 300;
+const flyoutHeight = 160;
 const flyoutMargin = 10;
```
Comment (ferhoyos) [inline frontend/src/components/TracingIntegration/TraceTooltip.tsx]: I consider flyout quite big, I would reduce the numbers a bit so you can see the rest of the traces graph:

Image

```suggestion
const flyoutWidth = 260;
const flyoutHeight = 100;
const flyoutMargin = 5;
```

File: frontend/src/components/TracingIntegration/TraceTooltip.tsx [hunk 3]
```diff
@@ -12,26 +12,45 @@ import { PFColors } from 'components/Pf/PfColors';
...
 const innerWidth = flyoutWidth - 2 * flyoutMargin;
 const innerHeight = flyoutHeight - 2 * flyoutMargin;
 
 const tooltipStyle = kialiStyle({
   color: PFColors.ColorLight100,
+  backgroundColor: PFColors.Color100,
+  borderRadius: '4px',
+  padding: '10px',
```
Comment (ferhoyos) [inline frontend/src/components/TracingIntegration/TraceTooltip.tsx]: padding, radius and margin should be in rem.

```suggestion
  borderRadius: '0.25rem',
  padding: '0.5rem',
```

File: frontend/src/components/TracingIntegration/TraceTooltip.tsx [hunk 4]
```diff
@@ -12,26 +12,45 @@ import { PFColors } from 'components/Pf/PfColors';
...
+  lineHeight: '1.4'
 });
 
 const titleStyle = kialiStyle({
   whiteSpace: 'nowrap',
   overflow: 'hidden',
-  textOverflow: 'ellipsis'
+  textOverflow: 'ellipsis',
+  fontWeight: 'bold',
+  marginBottom: '8px'
```
Comment (ferhoyos) [inline frontend/src/components/TracingIntegration/TraceTooltip.tsx]:
```suggestion
  marginBottom: '0.5rem'
```

File: frontend/src/components/TracingIntegration/TraceTooltip.tsx [hunk 5]
```diff
@@ -12,26 +12,45 @@ import { PFColors } from 'components/Pf/PfColors';
...
-const contentStyle = kialiStyle({ width: '100%', height: '100%' });
-const leftStyle = kialiStyle({ width: '35%', height: '100%', float: 'left' });
+const contentStyle = kialiStyle({
+  display: 'flex',
+  alignItems: 'center'
+});
+
+const leftStyle = kialiStyle({
+  width: '80px',
+  marginRight: '12px',
```
Comment (ferhoyos) [inline frontend/src/components/TracingIntegration/TraceTooltip.tsx]:
```suggestion
  marginRight: '0.75rem',
```

File: frontend/src/components/TracingIntegration/TraceTooltip.tsx [hunk 6]
```diff
@@ -12,26 +12,45 @@ import { PFColors } from 'components/Pf/PfColors';
...
 
 const tooltipStyle = kialiStyle({
   color: PFColors.ColorLight100,
+  backgroundColor: PFColors.Color100,
+  borderRadius: '4px',
+  padding: '10px',
   width: innerWidth,
-  height: innerHeight
+  height: innerHeight,
+  fontSize: '12px',
```
Comment (ferhoyos) [inline frontend/src/components/TracingIntegration/TraceTooltip.tsx]:
```suggestion
  fontSize: '0.75rem',
```

File: frontend/src/components/TracingIntegration/TraceTooltip.tsx [hunk 7]
```diff
@@ -90,15 +109,37 @@ const mapStateToProps = (
 const TraceLabelContainer = connect(mapStateToProps)(TraceLabel);
 
 export class TraceTooltip extends React.Component<HookedTooltipProps<JaegerLineInfo>> {
+  private getOrientation = (props: any): 'top' | 'bottom' | 'left' | 'right' => {
```
Comment (ferhoyos) [inline frontend/src/components/TracingIntegration/TraceTooltip.tsx]: Two comments here:

1. The new getOrientation method has 4 distinct code paths (right/left/bottom/top) with boundary conditions, but no Jest coverage. Existing TraceTooltip.test.tsx only tests TraceLabel and mocks out HookedChartTooltip. Since this logic drove the reported tooltip overflow bug, regressions are easy to reintroduce.  **Suggested**: Extract getOrientation as a pure exported function (e.g., computeFlyoutOrientation({x, y, width}, {flyoutWidth, flyoutHeight, flyoutMargin})) and add a table-driven test covering: near-left-edge -> 'right', near-right-edge -> 'left', width=0 -> skip left check, near-top -> 'bottom', default -> 'top', and boundary equality values.

2. getOrientation falls back to width = 0 when props.width is not a number. When width === 0, the right-edge guard (width > 0 && x > width - horizontalMargin) never fires, meaning tooltips near the right chart edge won't get 'left' orientation. In practice Victory should provide width, but if it doesn't, the tooltip could extend past the right edge. A defensive fallback or log would make this more robust.

File: frontend/src/components/TracingIntegration/TraceTooltip.tsx [hunk 8]
```diff
@@ -90,15 +109,37 @@ const mapStateToProps = (
...
       const trace = this.props.activePoints[0].trace;
       return (
         <HookedChartTooltip
           {...this.props}
+          constrainToVisibleArea={false}
           flyoutWidth={flyoutWidth}
           flyoutHeight={flyoutHeight}
-          flyoutComponent={<ChartCursorFlyout style={{ stroke: 'none', fillOpacity: 0.6 }} />}
+          orientation={this.getOrientation}
+          flyoutComponent={<ChartCursorFlyout style={{ stroke: 'none', fillOpacity: 0.9 }} />}
```
Comment (ferhoyos) [inline frontend/src/components/TracingIntegration/TraceTooltip.tsx]: When the tooltip appears, its flyout (ChartCursorFlyout) is rendered as an SVG <rect> on top of the scatter points. Since the flyout has no pointerEvents: 'none', it captures mouse events before they can reach the scatter circle underneath. So when you try to click a point whose tooltip is already showing, the click hits the flyout, not the data point.

The fix is straightforward — add pointerEvents: 'none' to the flyout style so clicks pass through to the data points underneath:

```suggestion
          flyoutComponent={<ChartCursorFlyout style={{ stroke: 'none', fillOpacity: 0.9, pointerEvents: 'none' }} />}
```

One question: Why have you increased opacity to 0.9? Opacity tooltips in Kiali charts are usually between 0.5 and 0.6 (e.g. SparklineChart, RateChart)

**Review by ferhoyos — CHANGES_REQUESTED** [review]
Thanks for fixing this regression bug @josunect! I hope this is the only layout refactoring bug we have. I have some comments to address.

File: frontend/src/components/TracingIntegration/TraceTooltip.tsx (hunk 2 above)
Comment (josunect) [reply frontend/src/components/TracingIntegration/TraceTooltip.tsx]: I don't think it should reduce the width and hide part of the data, the height and margin, can be reduced

**Comment by josunect** [comment]
@ferhoyos thanks for the review! I've done the suggested changes

File: frontend/src/components/TracingIntegration/TraceTooltip.tsx (hunk 2 above)
Comment (ferhoyos) [reply frontend/src/components/TracingIntegration/TraceTooltip.tsx]: I'm fine to maintain the width if you want to show more URL data.

File: frontend/src/components/TracingIntegration/__tests__/TraceTooltip.test.tsx
Comment (ferhoyos) [inline frontend/src/components/TracingIntegration/__tests__/TraceTooltip.test.tsx]: @josunect you have to rebase this PR from master and check if the unit test modification works since yesterday we migrated from Enzyme to RTL.

**Comment by ferhoyos** [comment]
@josunect thanks for addressing my feedback! Now everything works as expected. Just one comment about checking unit test modification with RTL.

File: frontend/src/components/TracingIntegration/__tests__/TraceTooltip.test.tsx
Comment (josunect) [reply frontend/src/components/TracingIntegration/__tests__/TraceTooltip.test.tsx]: @ferhoyos I've rebased and the test is working fine

**Review by ferhoyos — APPROVED** [review]
LGTM
